rotkehlchen/exchanges/okx.py

```diff
@@ -82,7 +82,7 @@ def edit_exchange_credentials(self, credentials: ExchangeAuthCredentials) -> boo
         if credentials.api_key is not None:
             self.session.headers.update({'OK-ACCESS-KEY': self.api_key})
         if credentials.passphrase is not None:
-            self.session.headers.update({'OK-ACCESS-PASSPHRASE': self.passphrase})
+            self.session.headers.update({'OK-ACCESS-PASSPHRASE': credentials.passphrase})
```

Comment:
this is a bug right? Please add a changelog entry for this. Also add one for coinbasePro. For ftx we had

```
* :feature:`5933` FTX and FTX US support is removed since the exchanges no longer exist
```